extensions/WEBGL_shader_pixel_local_storage/extension.xml

```diff
@@ -109,24 +135,75 @@ interface WEBGL_shader_pixel_local_storage {
   <newfun>
     <function name="framebufferPixelLocalClearValuefvWEBGL" type="undefined">
       <param name="plane" type="GLint"/>
-      <param name="values" type="Float32List"/>
+      <param name="value" type="sequence&lt;GLfloat&gt;"/>
+      Sets the floating point clear value for the given plane.
+      <div class="note">
+        If <code>value</code> has less than <code>4</code> elements, generates an
+        <code>INVALID_OPERATION</code> error.
+      </div>
+    </function>
+  </newfun>
+
+  <newfun>
+    <function name="framebufferPixelLocalClearValueivWEBGL" type="undefined">
+      <param name="plane" type="GLint"/>
+      <param name="value" type="sequence&lt;GLint&gt;"/>
+      Sets the integer clear value for the given plane.
+      <div class="note">
+        If <code>value</code> has less than <code>4</code> elements, generates an
+        <code>INVALID_OPERATION</code> error.
+      </div>
+    </function>
+  </newfun>
+
+  <newfun>
+    <function name="framebufferPixelLocalClearValueuivWEBGL" type="undefined">
+      <param name="plane" type="GLint"/>
+      <param name="value" type="sequence&lt;GLuint&gt;"/>
+      Sets the unsigned integer clear value for the given plane.
+      <div class="note">
+        If <code>value</code> has less than <code>4</code> elements, generates an
+        <code>INVALID_OPERATION</code> error.
+      </div>
+    </function>
+  </newfun>
+
+  <newfun>
+    <function name="framebufferPixelLocalClearValuefvWEBGL" type="undefined">
```

Comment:
Since these entry points take in only a small amount of data and aren't reading back values to JavaScript, the pattern in the WebGL 2.0 spec for the `uniform*` entry points seems more applicable - take in e.g. Float32List (e.g. either a Typed Array or a sequence of floats) and have the src_offset be an optional argument defaulting to 0. To save time, I'll merge this and propose a follow-on PR.